testing/tests/ui.rs

```diff
@@ -21,4 +23,27 @@ fn ui() {
     if rustc::is_max_version("1.53").unwrap() {
         t.compile_fail("tests/ui/before_1.54/*.rs");
     }
+
+    // To be able to use existing templates, we create a link to the `templates` folder.
+    let manifest_dir = match std::env::var("CARGO_MANIFEST_DIR") {
+        Ok(manifest_dir) => manifest_dir,
+        Err(_) => panic!("you need to run tests with `cargo`"),
+    };
+    let target = Path::new(&manifest_dir).join("../target/tests/trybuild/askama_testing");
+    if !target.exists() {
+        if let Err(err) = std::fs::create_dir_all(&target) {
+            panic!("Failed to create folder `{}`: {err:?}", target.display());
+        }
+    }
+    let target = target.canonicalize().unwrap().join("templates");
+    if !target.exists() {
+        let original = Path::new(&manifest_dir).join("templates");
+        if std::os::unix::fs::symlink(&original, &target).is_err() {
```

Comment:
Nit: please import `use std::os::unix::fs::symlink` at the top of the file.

Reply:
I will need to add a `cfg` for it so I'll import it in the function instead (unless you prefer a `cfg` on top of the `use`?),

Reply:
Maybe a separate commit that just hoists `#![cfg(not(windows))]` to the module level?

Reply:
Sounds good to me.